virtwho/config.py

```diff
@@ -1137,7 +1140,9 @@ def validate(self):
             # This next check will not be necessary after we know that all sections are
             # ConfigSections
```

Comment:
I'm sure we know that all sections are subclasses of `ConfiSections`. Thus following check can be removed.